cactusbot/commands/command.py

```diff
@@ -3,6 +3,14 @@
 import inspect
 import re
 
+ROLES = {
```

Comment:
Good question - basically, I wanted to leave it open for a potential "new" thing, be that regulars, or CactusDev subscribers, or... whatever. Also, 5 is a nice number. ;P